src/stores/ProductStore.ts

```diff
@@ -0,0 +1,27 @@
+import { makeAutoObservable } from "mobx";
+
+interface Product
+{
+    id: number;
+    name: string;
+    price: number;
+}
+
+class ProductStore
+{
+    products: Product[] = [
```

Comment:
should be observable. please pay more attention on comments given for other lines. comment should be applied on al the code

Reply:
Done